src/components/common/CompletingParty.vue

```diff
@@ -58,6 +60,18 @@ export default class CompletingParty extends Vue {
     return (this.validate && !this.completingPartyValid)
   }
 
+  // FUTURE: pull this from shared auth composable (after vue3 upgrade)
+  get isSbcStaff (): boolean {
+    const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+    if (!currentAccount) return false
+    try {
+      return JSON.parse(currentAccount).accountType === AccountTypes.STAFF_SBC
```

Comment:
Weird that you're not getting an error here with the wrong AccountTypes enum (STAFF_SBC vs SBC_STAFF).

Reply:
ah I switched the enum name name after testing it in the UI (I had it different in the create ui)